scripts/build-codesearch.py

```diff
@@ -77,7 +77,7 @@ def copy_objdir_files(dest_dir, config):
 
 json.dump(livegrep_config, open('/tmp/livegrep.json', 'w'))
 
-run(['codesearch', '/tmp/livegrep.json', '-dump_index', tree['codesearch_path'],
+run(['codesearch', '/tmp/livegrep.json', '-dump_index', tree['codesearch_path'], '-index_only',
```

Comment:
Is this just something that is needed for the new version?

Reply:
Yeah, otherwise it starts the server along with dumping the index. Previously I think the `-dump_index` automatically suppressed starting the server.